src/SessionSetup/OtherUserDialog.ui

```diff
@@ -0,0 +1,133 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ui version="4.0">
+ <class>OtherUserDialog</class>
+ <widget class="QDialog" name="OtherUserDialog">
+  <property name="geometry">
+   <rect>
+    <x>0</x>
+    <y>0</y>
+    <width>530</width>
+    <height>105</height>
+   </rect>
+  </property>
+  <property name="windowTitle">
+   <string>Warning</string>
+  </property>
+  <layout class="QGridLayout" name="gridLayout" rowstretch="1,0,0">
+   <item row="0" column="0">
+    <layout class="QHBoxLayout" name="horizontalLayout">
+     <property name="spacing">
+      <number>0</number>
+     </property>
+     <item>
+      <widget class="QLabel" name="label">
+       <property name="text">
+        <string>This instance is owned by </string>
```

Comment:
Super-Nit: I'm a little tiny bit missing why we are showing this dialog. Maybe something like this: `This instance is owned by another user "%s"`.

Reply:
done